src/index.js

```diff
@@ -61,7 +62,7 @@ export default function (CustomElement, opts) {
       });
     }
     render() {
-      return React.createElement(tagName, null, this.props.children);
+      return React.createElement(tagName, { style: this.props.style }, this.props.children);
```

Comment:
@joscha that's for objects.

@stevemao It's still in TC39 [stage 2](https://github.com/tc39/proposals). I'll add a test since it's common practice in React, but I'll note that there's no special behaviour necessary for this so we're essentially testing Babel here.